JavaScript Grocery List Project/js/app.js

```diff
@@ -1,22 +1,28 @@
+//Event handler for enable list
+document.addEventListener("DOMContentLoaded", initList);
 
-(function(){
+function initList(){
 
+    
     let listHolder = [];    
     
+    //get data from local storage
+    function getOldData(){    
+         let oldList = localStorage.getItem('listHolder');

```

Comment:
why let? why not const? do you have an understanding about variable types?

Reply:
Yes, Dada, I have an understanding about variable types. 
I am used let because of I am updating (**changing**) listholder array variable.